dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/process/GitHubPullRequestSender.java

```diff
@@ -53,19 +53,13 @@ public GitHubPullRequestSender(DockerfileGitHubUtil dockerfileGitHubUtil, Forkab
             totalContentsFound++;
             parent = ghContent.getOwner();
             parentRepoName = parent.getFullName();
-            // Refresh the repo to ensure that the object has full details
-            try {
-                parent = dockerfileGitHubUtil.getRepo(parentRepoName);
-                ShouldForkResult shouldForkResult = forkableRepoValidator.shouldFork(parent, ghContent, gitForkBranch);
-                if (shouldForkResult.isForkable()) {
-                    contentsShouldFork++;
-                    // fork the parent if not already forked
-                    ensureForkedAndAddToListForProcessing(pathToDockerfilesInParentRepo, parent, parentRepoName, ghContent);
-                } else {
-                    log.warn("Skipping {} because {}", parentRepoName, shouldForkResult.getReason());
-                }
-            } catch (IOException exception) {
```

Comment:
Hey @RaviTezu ! Thanks for your contribution! 

I just have a couple of questions:
* Have you taken a look to see if there are any other iterations of this in the codebase?
* Are we sure that removing the try/catch is safe here? I'm wondering if that was also covering some error cases for the other invocations here.

Reply:
Thanks for the reply @justinharringa 
- Yes, I looked for other instances if we are explicitly populating GHRepository.parent and didn't find any. 
- The try/catch block was not needed once the call `dockerfileGitHubUtil.getRepo` has been removed. It is the only call which can raise an IOException. So I removed it.

Reply:
Great! Thanks @RaviTezu! Did you happen to try this out to make sure that it was able to successfully fork a found repo?

Reply:
@justinharringa I didn't try it. I ran the unittests and they passed.